src/auth/integration-tests/tests/driver.rs

```diff
@@ -28,12 +28,24 @@ mod driver {
     #[cfg(all(test, feature = "run-byoid-integration-tests"))]
     #[tokio::test(flavor = "multi_thread", worker_threads = 1)]
     async fn run_workload_identity_provider_url_sourced() -> anyhow::Result<()> {
-        auth_integration_tests::workload_identity_provider_url_sourced().await
+        auth_integration_tests::workload_identity_provider_url_sourced(true)
+            .await
+            .unwrap();
```

Comment:
Here and below, use the try operator `?`.

`anyhow::Result` will handle any input, so we don't need to `map_err()` or anything.